gremlin-go/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/lyndonb-bq/tinkerpop/gremlin-go
+
+go 1.17
+
+require (
+	github.com/google/uuid v1.3.0
```

Comment:
please ensure that the libraries satisfies the Apache requirement for third party libraries. https://www.apache.org/legal/resolved.html#category-a